generators/addons/index.js

```diff
@@ -131,9 +131,18 @@ module.exports = class extends Generator {
                     }
 
                 }
+                if(this.options.ci !== undefined) {
+                    newPkgConfig.scripts.test = "./node_modules/.bin/jest --config ./config/jest.config.json"
```

Comment:
@StfBauer we might need to remove that if AC already did it in the voitanos presets

Reply:
https://github.com/Voitanos/jest-preset-spfx-react16/blob/master/postinstall/index.js confirmed, we need to remove it